src/Install/Console/InstallCommand.php

```diff
@@ -133,7 +133,8 @@ protected function install()
                     'host' => 'required',
                     'database' => 'required|string',
                     'username' => 'required|string',
-                    'prefix' => 'alpha_dash|max:10'
+                    'prefix' => 'alpha_dash|max:10',
+                    'port'   => 'required|integer|min:1|max:65535',
```

Comment:
I don't know about Laravel's database driver specifically but in some other implementations the port needs to be omitted if a socket is used. That's another reason to make the port number optional.